tests/gold_tests/tls_hooks/tls_hooks10.test.py

```diff
@@ -24,7 +24,10 @@
 Test different combinations of TLS handshake hooks to ensure they are applied consistently.
 '''
 
-Test.SkipUnless(Condition.HasProgram("grep", "grep needs to be installed on system for this test to work"))
+Test.SkipUnless(
+    Condition.HasProgram("grep", "grep needs to be installed on system for this test to work"),
```

Comment:
What systems don't have grep installed?  Shouldn't it fail if grep is not installed?

Reply:
I would guess it might happen in an atomic container. I did not add the grep test. It was already there. I am only adding the test for ALPN support in this case as we see this test fails on Centos and RHEL systems using the built-in OpenSSL library